changelog.d/3085.bugfix

```diff
@@ -0,0 +1 @@
+Make sure we replace the 'asnwer call' pending intent on ringing call notifications.
```

Comment:
```suggestion
Make sure we replace the 'answer call' pending intent on ringing call notifications.
```

Also this description is quite technical. For the user, what will be the impact of this change?